application/modules/settings/views/partial_settings_email.php

```diff
@@ -72,6 +72,15 @@ class="form-control simple-select">
                                    class="form-control"
                                    value="<?php echo get_setting('smtp_server_address', '', true); ?>">
                         </div>
+                        
+                        <div class="form-group">
+                            <label for="settings[smtp_mail_from]">
+                                <?php _trans('smtp_mail_from'); ?>
+                            </label>
+                            <input type="text" name="settings[smtp_mail_from]" id="settings[smtp_mail_from]"
```

Comment:
Should we not check if the input is a mail instead of a text ? We need to be sure that the text entered here is a mail address no ?

Reply:
You mean 

    <input type=email ...

Reply:
Yes but on the phone it’s hard to write tags haha ! 

I don’t know if there is other validator in the settings for mail but I guess. Maybe add this field to the fields validated for mails no ?